cmd/controller-manager/app/controllermanager.go

```diff
@@ -135,6 +153,11 @@ func NewControllerInitializers() map[string]InitFunc {
 	return controllers
 }
 
+// KnownControllers returns all known controllers's name
+func KnownControllers() []string {
+	return sets.StringKeySet(NewControllerInitializers()).List()
```

Comment:
Should we sort the controllers? I'm afraid the list is unordered(the map is unordered).

Reply:
The `List` method will sort the results.

Reply:
Oh, thanks. Good to know.